.github/release-please/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
   "core": "24.28.0",
   "prover": "16.5.0",
-  "zk_toolbox": "0.1.2"
-}
+  "zkstack_cli": "0.1.2"
+}
```

Comment:
```suggestion
}

```

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3003/commits/92f807985b502f3502f10713716dd82d3d2f9cc5